paper-ripple-behavior.html

```diff
@@ -79,9 +79,11 @@
         }
         var domContainer = rippleContainer === this.shadyRoot ? this :
           rippleContainer;
-        if (opt_triggeringEvent &&
-            domContainer.contains(opt_triggeringEvent.target)) {
-          this._ripple.uiDownAction(opt_triggeringEvent);
+        if (opt_triggeringEvent) {
+          var target = opt_triggeringEvent.target;
+          if (domContainer.contains(/** @type {Node} */(target))) {
```

Comment:
Not easily. Event targets have the delightfully over-general type of EventTarget (because stuff like XHRs also dispatch events): https://github.com/google/closure-compiler/blob/master/externs/browser/w3c_event.js

I don't see a simple local improvement here.